server/surveyRdb/schemaRdb/dataColProps.js

```diff
@@ -54,6 +60,22 @@ const props = {
     [colTypeProcessor]: () => () => sqlTypes.decimal,
   },
 
+  [nodeDefType.date]: {
+    [colTypeProcessor]: () => () => sqlTypes.date,
+    [colValueProcessor]: (surveyInfo, nodeDefCol, nodeCol) => {
+      const [day, month, year] = Node.getNodeValue(nodeCol, '').split('/').map(trim)
```

Comment:
are the values in the db saved as dd/mm/yyyy ? I would have saved them as yyyy-mm-dd